workshop/watson-knowledge-catalog-admin/README.md

```diff
@@ -44,7 +44,12 @@ If the `Watson Knowledge Catalog` service instance is not added then click `Add`
 
 ![add local files to catalog](../.gitbook/assets/images/wkc-admin/wkc-add-to-catalog-local-files.png)
 
-* Browse to the `/data/split/applicant_personal_data.csv` file and double-click or click `Open`. Add an optional description and click `Add`:
+* Download the `application_personal_data.csv` file from [here](https://raw.githubusercontent.com/IBM/credit-risk-workshop-cpd/master/data/split/applicant_personal_data.csv). If 
```

Comment:
we should use the workshop branch, and use this link, letting github generate the actual raw link in case they change that logic.
```suggestion
* Download the `application_personal_data.csv` file from [here](https://github.com/IBM/credit-risk-workshop-cpd/raw/workshop-DDC/data/split/applicant_personal_data.csv). If 
```